langchain/vectorstores/redis.py

```diff
@@ -84,6 +84,33 @@ def add_texts(
     def similarity_search(
         self, query: str, k: int = 4, **kwargs: Any
     ) -> List[Document]:
+        docs_and_scores = self.similarity_search_with_score(query, k=k)
+        return [doc for doc, _ in docs_and_scores]
+
+    def similarity_search_limit_score(
+        self, query: str, k: int = 4, score_thread: float = 0.2, **kwargs: Any
+    ) -> List[Document]:
+        docs_and_scores = self.similarity_search_with_score(query, k=k)
+
+        docs = []
+        for doc in docs_and_scores:
```

Comment:
`for doc, score in docs_and_scores:` more readable

Reply:
Okay, it has been modified